Sources/SmileID/Classes/DocumentVerification/View/DocumentConfirmationView.swift

```diff
@@ -0,0 +1,52 @@
+import SwiftUI
+
+struct DocumentConfirmationView: View {
+    @ObservedObject var viewModel: DocumentCaptureViewModel
+    @EnvironmentObject var navigationViewModel: NavigationViewModel
+
+    var body: some View {
+        VStack(spacing: 49) {
+            VStack(spacing: 16) {
+                Text(SmileIDResourcesHelper.localizedString(for: "Document.Confirmation.Header"))                    .multilineTextAlignment(.center)
+                    .font(SmileID.theme.header2)
+                    .foregroundColor(SmileID.theme.accent)
+                    .lineSpacing(0.98)
+                Text(SmileIDResourcesHelper.localizedString(for: "Document.Confirmation.Callout"))                    .multilineTextAlignment(.center)
+                    .font(SmileID.theme.header5)
+                    .foregroundColor(SmileID.theme.tertiary)
+                    .lineSpacing(1.3)
+            }
+
+            Image(uiImage: viewModel.frontImage ?? UIImage())
```

Comment:
We should never get to a point where the image to display is unavailable.

Tying back to my previous comment, this component should be parametrized to take things like title, subtitle, image, etc as parameters

Reply:
Bump